features/logging/4.x/cluster_logging_operator.feature

```diff
@@ -0,0 +1,60 @@
+@clusterlogging
+Feature: cluster-logging-operator related test
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-21907
+  @admin
+  @destructive
+  Scenario: Deploy elasticsearch-operator via OLM using CLI
+    Given logging operators are installed successfully
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22492
+  @admin @destructive
+  Scenario: Scale Elasticsearch nodes by nodeCount 2->3->4 in clusterlogging
+    Given I switch to cluster admin pseudo user
+    And I create clusterlogging instance with:
+      | remove_logging_pods | true                                                                                                     |
+      | crd_yaml            | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/scalebase.yaml |
+      | log_collector       | fluentd                                                                                                  |
+    Then the step should succeed
+    Given I get project clusterlogging named "instance" as YAML
+    Then the step should succeed
+    And the output should contain:
+      | nodeCount: 2 |
+    Given 2 pods become ready with labels:
+      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
+    When I run the :patch client command with:
+      | resource      | clusterlogging                                          |
+      | resource_name | instance                                                |
+      | p             | {"spec":{"logStore":{"elasticsearch":{"nodeCount":3}}}} |
+      | type          | merge                                                   |
+    Then the step should succeed
+    Given I get project clusterlogging named "instance" as YAML
+    Then the step should succeed
+    And the output should contain:
+      | nodeCount: 3 |
+    And I wait up to 600 seconds for the steps to pass:
+    """
+    And 3 pods become ready with labels:
+      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
+    """
```

Comment:
why do you wrap into a wait step? `3 pods become ready with labels:` is already waiting